here_for_change/municipalities/views.py

```diff
@@ -10,3 +11,37 @@ class WardDetail(DetailView):
     model = Ward
     slug_field = 'slug__iexact'
     slug_url_kwarg = 'slug'
+    page_content = {
+        'ward': {
+            'councillor': {
+                'name': {
+                    'value': 'Danny Europa',
```

Comment:
@Hazmid so here is where `content.councillor.name.value` is set and pulled from.